### DISCUSSION TITLE
Why does supabase.auth.api.resetPasswordForEmail(email) return a 404

### DISCUSSION THREAD

**Discussion by robhicks** [supabase/supabase#1210] [Questions] [answered]
I am attempting to use supabase with [svelte kit](https://kit.svelte.dev/).

In server code, I make the upabase.auth.api.resetPasswordForEmail(email)  when a users wants to reset the user's password. The call succeeds because an email is sent. But the response I'm seeing from the request is 404. I'm wondering why.

**Comment by kiwicopple** [comment, marked as answer]
I can't think of any reason off the top of my head. Can you provide some code which can replicate this - if it's a bug we'd love to get it fixed!

**Reply by robhicks** [reply]
I have isolated the problem to svelte kit. The svelte kit endpoint is called and the supabase call succeeds but svelte kit then returns a 404.
